packages/cozy-client/src/models/bitwarden.js

```diff
@@ -0,0 +1,26 @@
+import { Q } from '../queries/dsl'
+import CozyClient from '../CozyClient'
```

Comment:
If this file is only imported for the JSDoc, then you should remove it and use the jsdoc import that way: 

```
   * @param {import("../types").CozyClientDocument} document - Document to query
```

for instance.

Reply:
I've updated all the imports in the JSDOC for CozyClient